.gitignore

```diff
@@ -1,36 +1,37 @@
 # Temporary generated files
-_out
+/_out
```

Comment:
why the whole file is prefixed with `/` ?
best to just remove the lines we don't need related to goimport in this PR please?

Reply:
I did it so we won't unknowingly won't miss any accidentally matched files. Why is it better to keep this limited to the one dependency that we just happened to find broken?

Reply:
looking again, you are right
thanks